### PR TITLE
run each Expr(toplevel) argument in the newest world

### PR DESCRIPTION
**Pull request by vtjnash**
This should fix the coverage submission: https://build.julialang.org/builders/coverage_ubuntu14.04-x64/builds/859/steps/Gather%20test%20results%20and%20Submit/logs/stdio

### CODE REVIEW

**Comment by tkelman** [comment]
Is there something within base that could test this?

**Comment by stevengj** [comment]
See how I tested #19784 for example.

File: test/parse.jl [hunk 1]
```diff
@@ -414,6 +414,26 @@ test_parseerror("0x1.0p", "invalid numeric constant \"0x1.0\"")
               try = "No"
            """)) == Expr(:error, "unexpected \"=\"")
 
+# issue #19861 make sure macro-expansion happens in the newest world for top-level expression
+@test eval(Base.parse_input_line("""
```
Comment (tkelman) [inline test/parse.jl]: would `include_string` be any better?

Comment (vtjnash) [inline test/parse.jl]: nah, the `include_string` code-path was fine. this is an an entirely different bit of code.

**Pull request merged**